conmon-rs/server/src/rpc.rs

```diff
@@ -58,7 +58,11 @@ impl conmon::Server for Server {
         let id = pry!(req.get_id()).to_string();
         debug!("Got a create container request for id {}", id);
 
-        let log_path = pry!(req.get_log_path());
+        let log_path = pry!(pry_err!(pry!(req.get_log_drivers())
+            .iter()
+            .next()
+            .context("no log driver provided"))
+        .get_path());
```

Comment:
I expect that we create a higher level logging structure for parsing this which will be then used for writing and passing down the information to the lower level loggers.